### PR TITLE
✅ Fix test error in Windows for `jsonable_encoder`

### PR DESCRIPTION
**Pull request by iudeen**
Fixes issue in this comment: https://github.com/tiangolo/fastapi/pull/9816#discussion_r1257415632

### CODE REVIEW

**Comment by iudeen** [comment]
Question here is, can we make other tests also follow the same pattern of using `str(Path)`? I feel we don't have to hardcode the output of `str(Path)` in our tests.

File: tests/test_jsonable_encoder.py [hunk 1]
```diff
@@ -247,8 +247,11 @@ class ModelWithPath(BaseModel):
             class Config:
                 arbitrary_types_allowed = True
 
-    obj = ModelWithPath(path=PurePath("/foo", "bar"))
-    assert jsonable_encoder(obj) == {"path": "/foo/bar"}
+    test_path = PurePath("/foo", "bar")
+
+    obj = ModelWithPath(path=test_path)
+
+    assert jsonable_encoder(obj) == {"path": str(test_path)}
```
Comment (Kludex) [inline tests/test_jsonable_encoder.py:254]:
```suggestion
    test_path = PurePath("/foo", "bar")
    obj = ModelWithPath(path=test_path)
    assert jsonable_encoder(obj) == {"path": str(test_path)}
```

**Review by Kludex — APPROVED** [review]

**Review by yezz123 — APPROVED** [review]

**Comment by iudeen** [comment]
Can this also be merged? I don’t think this needs a release 🤔

**Comment by tiangolo** [comment]
Great, thank you @iudeen! 🚀

**Pull request merged**